src/Core/Worker/Factory/Process.php

```diff
@@ -8,28 +27,22 @@
 
 class Process extends BaseFactory implements ICanCreateFromTableRow
 {
+	public function determineHost(?string $hostname = null): string
+	{
+		if (empty($hostname)) {
+			$hostname = php_uname('n');
+		}
+
+		return strtolower($hostname);
```

Comment:
This is fine but what about a one liner?
```suggestion
		return strtolower($hostname ?? php_uname('n'));
```